static/src/javascripts/projects/facia/modules/onwards/search-tool.js

```diff
@@ -243,9 +243,8 @@ export class SearchTool {
         this.$input.val(inputValue);
     }
 
-    renderList(results: Array<City>, numOfResults: number): void {
+    renderList(results: Array<City>, resultsToShow: number): void {
         const docFragment = document.createDocumentFragment();
-        const resultsToShow = results.length - numOfResults;
```

Comment:
Struggling to understand purpose of this line.

Reply:
Weird, so it's currently saying don't show the last 5 items of the result set, so if there are 5 or less items it'll never show any of them? That doesn't seem right.

Reply:
Yeah - it returns a negative, which results in nothing being shown.

Reply:
So bizarre. It would be great to write some tests for this method, to indicate how it's supposed to be used!